docs/networks/l15-testnet.md

```diff
@@ -1,5 +1,8 @@
 ---
 sidebar_position: 3
+id: l15
```

Comment:
no need to add ids - by defaults it goes to the filename of the file. So here changing this will change the URL of the page, it will have an impact and SEO etc.

Can you please remove this?

Reply:
removed